docs/appnotes/0009-storage-label-format.md

```diff
@@ -0,0 +1,97 @@
+# Storage label format specification
+
+## Abstract
+
+Users of TAMS have requested guidance on the use of `get_url` labels on flow segments.
+People want these to provide information to aid the selection of URLs where multiple are available.
+This Application Note provides a specification for the format of `get_url` labels to address these needs.
+
+In terms of requirements, we primarily want to enable the choice of get_urls based on properties such as bandwidth, latency, and cost.
+Unfortunately, many of these cannot be meaningfully signalled in a direct and universal way.
+If for no other reason, they are largely dependant on the intermediate infrastructure between the server and client.
+This will vary based on the physical and logical location of the client as much as the server.
+We must, therefore, provide enough information that the client can infer these properties themselves.
+This may be done by providing information about the location, and the type of storage.
+
+Unfortunately there is no one shared naming convention for resource locations across cloud providers.
+There are shared properties (regions, and availability zones), but these aren't universally available.
+Some cloud providers don't provide availability zones.
```

Comment:
How about:
some cloud providers don't provide availability zones or have services that run at a regional level so don't need the availability zone to be specified.